docs/runtimes/function.md

```diff
@@ -104,6 +104,13 @@ $ serverless invoke -f <function-name>
 "Hello world"
 ```
 
+To invoke function locally
+
+```bash
+$ serverless invoke local -f <function-name>
+"Hello world"
+```
```

Comment:
This part is covered by this page: https://bref.sh/docs/local-development.html

I think we can get rid of that.